libfwupdplugin/fu-volume.c

```diff
@@ -647,7 +656,9 @@ fu_volume_new_by_devnum(guint32 devnum, GError **error)
  * fu_volume_new_esp_default:
  * @error: (nullable): optional return location for an error
  *
- * Gets the platform default ESP
+ * Gets the platform default ESP.
+ *
+ * NOTE: This has been deprecated, please use fu_context_get_esp_volumes() instead.
```

Comment:
Didn't we need to add this to a list somewhere to say we need to drop it on the next ABI change?

Reply:
Indeed! Added to `readme.md` alongside the others.